examples/reown/src/utils/appkit.ts

```diff
@@ -0,0 +1,43 @@
+import { ChainType, type ExtendedChain } from '@lifi/widget'
+import type { AppKitNetwork, ChainNamespace } from '@reown/appkit-common'
+import { defineChain } from '@reown/appkit/networks'
+
+export type AppkitsupportedChainTypes = Exclude<ChainType, ChainType.MVM>
```

Comment:
small typo, `AppKitSupportedChainTypes`?

Reply:
Also, it might be better to only include supported AppKit chains to avoid updating this type when new ecosystems arrive.